frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -354,7 +354,13 @@ export type PrometheusActivityProps = {
   results: PrometheusResponse[];
 };
 
+export type alertType = {
+  alertMessage: string;
+  troubleshootLink?: string;
+};
+
 export type SubsystemHealth = {
+  alerts?: alertType[];
```

Comment:
Why are we using this? I am confused?

Reply:
https://github.com/openshift/console/pull/9426/files#diff-878da217ade003af00a546ddc59013b7674fe3f9d905813f541733719a9012b3R13

I've made this PR much simpler than before in the next commit. Pushing in a bit.